### PR TITLE
[WNMGDS-1142] Clean up files and READMEs for ds-medicare-gov

### PR DESCRIPTION
**Pull request by pwolfert**
https://jira.cms.gov/browse/WNMGDS-1142

## Summary
For context, see https://github.com/CMSgov/design-system/pull/1212

### Changed
- Delete unnecessary files
- Update READMEs

## How to test
Take a look at the READMEs and make sure the commands to build and test still work.

### CODE REVIEW

**Review by forrestbaer — APPROVED** [review]
Tests & Build passed, looks good

**Review by zarahzachz — CHANGES_REQUESTED** [review]
I don't know if this is considered part of this work, but there are some links on both MC and HC that point to outdated URLs:
1. Getting Started > Installation > #Need help or ran into an issue? has a broken link that I assume wasn't interpolated correctly
2. Getting Started > Child Design Systems > #Existing Child Design Systems points to deprecated repos

**Review by zarahzachz — COMMENTED** [review]
I was able to run all the new commands without issue

File: README.md [hunk 1]
```diff
@@ -43,23 +45,31 @@ These scripts can all be run from the root level of the repo:
...
-  - Publishes npm packages. Read our [Release Process guide](/guides/RELEASE-PROCESS.md) for more info.
+  - `yarn build:healthcare` to build the Healthcare.gov Design System
+  - `yarn build:medicare` to build the Medicare.gov Design System
 - `yarn test`
   - Runs JS unit tests
   - Runs Prettier for formatting
   - Lints JS using ESLint
   - Lints Sass using stylelint
+- `yarn test:unit`
+  - Runs only the JS unit tests
```
Comment (scheul93) [inline README.md:60]:
```suggestion
  - Runs JS unit tests for all packages
```

File: README.md [hunk 2]
```diff
@@ -43,23 +45,31 @@ These scripts can all be run from the root level of the repo:
...
 - `yarn test:a11y`
-  - Runs end to end tests
+  - Runs acessibility tests
```
Comment (scheul93) [inline README.md:62]:
```suggestion
  - Runs accessibility tests for design-system package only
```

**Comment by pwolfert** [comment]
@zarahzachz, I was digging into this

> 1. Getting Started > Installation > #Need help or ran into an issue? has a broken link that I assume wasn't interpolated correctly

and realized that you actually found a core docs site bug. It exists on the base CMSDS docs site too! https://design.cms.gov/startup/installation/

File: packages/design-system-docs/src/pages/startup/child-design-systems.md [hunk 3]
```diff
@@ -33,6 +33,5 @@ See the [child design system example](https://github.com/CMSgov/design-system/tr
...
 
-- A child design system should be actively maintained by product teams using the child design system. The core CMSDS team is not responsible for updating child design systems, however we are responsible for guidelines and assisting with training and adoption.
-- Users of child design systems should contribute back to the larger CMS design system community by filing issues/bugs in Github or Jira, giving feedback, and general maintenance. Collaboration and participation are essential to the success of the CMS design system family.
-- A child design system should defer to the core CMS design system for best practices or accessibility guidelines.
+- The healthcare.gov and medicare.gov child design systems are actively maintained by the core CMSDS team. Please reach out if you have questions about adoption, use, contribution, or anything else related to these design systems.
+- Users of child design systems are encouraged to contribute back to the larger CMS design system community by filing issues/bugs in Github or Jira, giving feedback, and general maintenance. Collaboration and participation are important to the success of the CMS design system family.
```
Comment (pwolfert) [inline packages/design-system-docs/src/pages/startup/child-design-systems.md:37]: Everyone, please look at the changes I made here and let me know what you think. CC @line47

**Review by zarahzachz — APPROVED** [review]

**Review by scheul93 — APPROVED** [review]

**Pull request merged**